setup.py

```diff
@@ -3,8 +3,30 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-with open("mplc/requirements.txt", "r") as fh:
-    requirements = fh.read()
+requirements = [
+    'h5py==2.10.0',
+    'joblib==1.0.0',
+    'librosa==0.8.0',
+    'matplotlib==3.3.3',
+    'numpy==1.19.4',
+    'scipy==1.5.4',
+    'scikit-learn==0.23.2',
+    'pandas==1.1.5',
+    'seaborn==0.11.0',
+    'loguru==0.5.3',
+    'tensorflow==2.4.1',
+    'ruamel.yaml==0.16.12',
+    'flake8==3.8.3',
+    'setuptools==50.3.0',
+    'wheel==0.35.1',
+    'twine==3.2.0'
```

Comment:
twine is only for the publication of the package on pypi servers, I wouldn't add it here, no?

Reply:
Should I extract all dev/packaging dependencies in a custom install ?

Reply:
I think that having the dev-requirements.txt in a specific file is fine as it is only used for packaging and publishing new builds